includes/library/zencart/QueryBuilderDefinitions/src/formatters/AbstractFormatter.php

```diff
@@ -134,7 +134,7 @@ public function getFormattedTotals($outputLayout)
               $total = 0; 
               if ($this->outputLayout['fields'][$key]['total'] == 'currencySum') {
                  foreach ($this->formattedResults as $rec) { 
-                    $total += preg_replace("/([^0-9\\.])/i", "", $rec[$key]); 
+                    $total += preg_replace("/([^0-9\.])/i", "", $rec[$key]); 
```

Comment:
FYI, inside the square brackets of a regex pattern like this the only chars that would need escaping would be `]` (because it ends the square-bracket pattern), or a `-` if it isn't the last character (because it might be confused with specifying a character range like a-z etc).  So technically the `.` doesn't need the `\` in this case.